kobo/apps/superuser_stats/admin.py

```diff
@@ -114,3 +196,4 @@ def __get_serialized_data(self, request) -> list:
 
 
 admin.site.register(KobocatSubmissionCounter, UserStatisticsAdmin)
+admin.site.register(ReadOnlyKobocatInstance, SubmissionsByCountry)
```

Comment:
We should deactivate the add button. These reports are read-only, aren't they?

![Screen Shot 2021-07-01 at 11 06 22](https://user-images.githubusercontent.com/24226825/124147929-2a371a00-da5d-11eb-810f-be8b6694508c.png)

![Screen Shot 2021-07-01 at 11 10 28](https://user-images.githubusercontent.com/24226825/124147930-2a371a00-da5d-11eb-8687-50128e46f8ef.png)